packages/connect-explorer/src/js/data/methods/management/getFeatures.ts

```diff
@@ -0,0 +1,11 @@
+const name = 'getFeatures';
+const docs = 'methods/getFeatures.md';
```

Comment:
It looks like this variable `docs` is never used.

Reply:
removed. docs does not exist in https://github.com/trezor/connect/tree/develop/docs/methods